### ISSUE TITLE
UI as a separate package

### ISSUE THREAD

**Issue by DeveloLongScript** [Xyphyn/photon#470]
Hey. Recently, I saw this project, and wondered what UI framework it was using. Obviously, its using Mono UI, something you made. Personally, I'm a React developer and I really appreciate good UI, and Photon has great UI, probably better UI then some other companies.

I was wondering if you ever thought about starting to maintain a UI library like Mono, or if thats something that you think is never gonna happen. Looks like you've done it, but then deprecated it and merged it into Photon. I just want you to know that your UI is great, and I'd love to see an implementation in React or Svelte, I could definitely use this in some of my projects.

Always thought of this quote:
> I personally believe that good design contributes significant value to software. It's one of the main methods of establishing trust between you and an internet stranger. Trust is important for internet businesses because it is the first thing a visitor evaluates before pulling out their credit card and becoming a customer.
> Some questions visitors might ask themselves are:
> - "Is this company legit?"
> - "Who else is using it?"
> - "Can I trust them with my personal data?"

from [MagicUI](https://magicui.design/docs)

and I think the UI that you created perfectly fits why Photon is such a great project

**Comment by DeveloLongScript** [comment]
**TL;DR** I'd love to see a revival of the existing Mono UI that maybe could be compatible with both React and Svelte

**Comment by DeveloLongScript** [comment]
I've decided to start porting mono-ui to React.
